schematic/store/synapse.py

```diff
@@ -413,7 +413,7 @@ def getFilesInStorageDataset(
         """
 
         # select all files within a given storage dataset folder (top level folder in a Synapse storage project or folder marked with contentType = 'dataset')
-        walked_path = walk(self.syn, datasetId)
+        walked_path = walk(self.syn, datasetId, includeTypes=["folder", "file"])
```

Comment:
Nit: I like to be explicit and do synapseutils.walk because there is an os.walk.

Reply:
Sounds good. Thanks for the suggestion Tom.